default/app.conf

```diff
@@ -16,4 +16,4 @@ version = 1.0.2
 
 [package]
 id = TA-cisco_phishing
-check_for_updates = true
+check_for_updates = false
```

Comment:
I assume this app isn't on Splunkbase, which is why it shouldn't check for updates?

Reply:
it is not on splunkbase